package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/single-spa-vue.js",
   "scripts": {
     "prepublishOnly": "yarn build",
-    "build": "rimraf lib && babel src --out-dir lib --source-maps --ignore '**/*.test.js'",
+    "build": "rimraf lib && rollup -c",
```

Comment:
Using rollup instead of babel cli because we now have our first 3p dependency (`css.escape`) and want it bundled into the output file so that single-spa-vue.js can be a standalone UMD file that can be used via script tag.